ably.d.ts

```diff
@@ -698,7 +699,7 @@ export interface TokenParams {
    *
    * @defaultValue `'{"*":["*"]}'`
    */
-  capability?: { [key: string]: capabilityOp[] } | string;
+  capability?: { [key: string]: capabilityOp[] | ['*'] } | string;
```

Comment:
Shouldn't `*` value be added to `capabilityOp` type above?
Based on the new empty line added near that type in this PR, is there a chance you already tried it and something went wrong?

Reply:
Also, OOI, is this the correct list of all capabilities available for token params: https://ably.com/docs/auth/capabilities? I couldn't find the list in the spec, only in the docs on a website.
If that list is correct, it appears that we're also missing `privileged-headers` capability type. If there's nothing special about it, maybe we should add it too and expand this PR a bit

Reply:
> Shouldn't * value be added to capabilityOp type above?

We could achieve the same result that way, yes! The reason I thought it would be worth making it separate is because if you specify `*`, you're essentially making any other specified capabilities redundant. So it doesn't really make sense to have `*` with some other `capabilityOp`. Happy to go with whatever you'd prefer, though!

> Also, OOI, is this the correct list of all capabilities available for token params

Yes that's the correct list, we could definitely add `privileged-headers` - which I'll do now

Reply:
> We could achieve the same result that way, yes! The reason I thought it would be worth making it separate is because if you specify *, you're essentially making any other specified capabilities redundant. So it doesn't really make sense to have * with some other capabilityOp. Happy to go with whatever you'd prefer, though!

Oh, I see now what you did there. Cool stuff! I'm happy to leave it as `capabilityOp[] | ['*']` then.